pkg/registration/spoke/bootstrapkubeconfigsmanager/bootstrapkubeconfiginuse.go

```diff
@@ -0,0 +1,53 @@
+package bootstrapkubeconfigsmanager
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// bootstrapKubeConfigInUse is the registration spoke's current in used bootstrap kubeconfig.
+type bootstrapKubeConfigInUse interface {
```

Comment:
ack.

Reply:
No, the current bootstrap kubeconfig used to trigger rebootstrap is based on the secret, and if in the future we want to change to another type such as file, then the interface is a good practice to prevent that change impact other parts of code.